doc/develop/dataio.rst

```diff
@@ -104,6 +105,48 @@ To read data from an OME-TIFF file, one could write:
    print(das[0].metadata[model.MD_LENS_MAG])
    print(das[0].metadata)
 
+Image position metadata
+=======================
+Most of the data store correspond to "spatial" data, i.e., an image representing
+the with axes in X and Y. In order to ensure a perfect overlay between data from
+different acquisition type, the metadata describes precisely how the image should
+be positioned. The following metadata has influence:
+
+* ``MD_PIXEL_SIZE``: size of a pixel (in m) in X and Y. In other words, this is the scale.
+* ``MD_POS``: the position (in m) of the *center of the image* in X and Y
+* ``MD_ROTATION``: counter-clockwise rotation (in radians) applied to the image from its center
+* ``MD_SHEAR``: *vertical* shear
+
+All values default to 0, excepted for the ``MD_PIXEL_SIZE`` which is required.
+
+Note that during acquisition, all these metadata have a twin-brother named with an extra
+`_COR` in order to record the correction on the image display. The function ``util.img.mergeMetadata()``
+can be used to merge these corrections into the main metadata. Before saving data
+to a file, the correction is automatically merged. So typically, after opening a
+file, the data will not have any of these extra correction metadata.
+
+For a given pixel situated at coordinates *p = i, j* in an image, its "physical" position
+*P = x, y* (in meters) can be computed with the following formula *P = RSLp + T*,
+where R is the rotation, L is the scale, and S is the shear.
+
+In Python, this can be done with:
+
+.. code-block:: python
+
+   from odemis import model
+   from odemis.util.transform import AffineTransform
+   pxs = md.get[model.MD_PIXEL_SIZE]
+   translation = md.get(model.MD_POS, (0, 0))
+   rotation = md.get(model.MD_ROTATION, 0)
+   shear = md.get(model.MD_SHEAR, 0)
+
+   tform = AffineTransform(rotation=rotation, scale=pxs, translation=translation)
+   L = numpy.array([(1, 0), (-shear, 1)])
+   tform.transformation_matrix = numpy.dot(tform.transformation_matrix, L)
```

Comment:
why add the negative shear, and not just add the shear as an argument to the class?

Reply:
That's what took a full day of debugging when we implemented it in Odemis! The AffineTransform implements an horizontal shear, however, in Odemis (overlay calibration), it turns out that we had a vertical shear (and I think that the reason is that we had just picked something random, although for the typical case, it would have made sense to use a horizontal shear).
The simplest way we've found (with Victoria) to work-around this discrepancy is this separate shear afterwards.